doc/input-output-reference/src/overview/group-location-climate-weather-file-access.tex

```diff
@@ -1819,7 +1819,7 @@ \subsubsection{Inputs}\label{inputs-22-002}
 
 \subsection{RoofIrrigation}\label{roofirrigation}
 
-The RoofIrrigation object is used to describe the amount of irrigation on the ecoroof surface over the course of the simulation runperiod. This object is used to provide irrigation data using Schedule objects that define rates of irrigation in meters per hour. These schedules can be one of two types: Schedule, or SmartSchedule.
+The RoofIrrigation object is used to describe the amount of irrigation on the ecoroof surface over the course of the simulation runperiod. This object is used to provide irrigation data using Schedule objects that define rates of irrigation in meters per hour. These schedules can be one of two types: Schedule, or SmartSchedule. Up till version 9.6, the schedule objects are necessary to describe the rates of rainfall. For newer versions, if the schedule is not defined, precipitation depth in the weather input .epw will be used instead. When this is the case, please make sure the precipitation in the .epw is accurate. This \href{table by the State Climate Extreme Committee}{https://www.ncdc.noaa.gov/extremes/scec/records/all/maxp} could be a reference for sanity check of extreme values in the precipitation data. The tabular report "Monthly Precipitation Summary" provides a monthly summary of rainfall in the weather file and the amount used in the roof irrigation. This could be handing in debugging the precipitation data in the weather file.
```

Comment:
There is a typo, the "Up till version 9.6, the schedule objects are necessary to describe the rates of rainfall" should be "... the rates of irrigation". This might have caused some confusion. 

I agree that there might be some different interpretation and users might want to include the rain fall as part of the actual irrigation. I added a "Use Precipitation" field to RoofIrrigation object. When it is "YES", the following will happen:                 
state.dataWaterData->Irrigation.ActualAmount = state.dataEcoRoofMgr->CurrentIrrigation + state.dataEcoRoofMgr->CurrentPrecipitation;

It will show up in the next push. I still need to fix some unit test.